resources/install.rb

```diff
@@ -34,16 +35,22 @@
     # enable powertools repo for libedit-devel
     include_recipe 'osl-repos::alma'
 
-    # use Remi PHP module to override stock php
-    # programatically define resource as to not have a bit long case/when
-    declare_resource(:"yum_remi_php#{shortver}", 'default')
+    if new_resource.remi_packages
+      declare_resource(:yum_remi_safe, 'default')
+    else
+      declare_resource(:"yum_remi_php#{shortver}", 'default')
+    end
   end
 
   # install default packages if no packages were specified, but wait to select the mod_php and pear packages
   if all_packages.empty? && all_php_packages.empty?
     all_php_packages = osl_php_default_installation_packages_without_prefixes
   end
 
+  if new_resource.remi_packages
+    prefix = "php#{shortver}-php"
+  end
```

Comment:
Let's go ahead and move this above the `all_packages` block just above here to ensure this gets set in the correct spot.
```suggestion
  # Determine package prefix based on installation type
  prefix = if new_resource.remi_packages
    "php#{shortver}-php"
  else
    'php'
  end
```